tests/unit/amo/pages/TestSearchPage.js

```diff
@@ -125,42 +125,6 @@ describe(__filename, () => {
     sinon.assert.callCount(fakeDispatch, 1);
   });
 
-  it('dispatches a server redirect when `atype` parameter is "4"', () => {
-    const fakeDispatch = sinon.spy(store, 'dispatch');
-
-    render({
-      location: createFakeLocation({ query: { atype: 4 } }),
-      store,
-    });
-
-    sinon.assert.calledWith(
-      fakeDispatch,
-      sendServerRedirect({
-        status: 301,
-        url: '/en-US/android/search/?type=search',
-      }),
-    );
-    sinon.assert.callCount(fakeDispatch, 1);
-  });
```

Comment:
I think we need to keep this test case.

Reply:
Can you take a look at this error please @willdurand 

```
 FAIL  tests/unit/amo/pages/TestSearchPage.js
  ● Console

    console.info node_modules/pino/browser.js:188
      [GA] GA disabled because trackingEnabled was false
    console.info node_modules/pino/browser.js:188
      `operatingSystem` filter is set to "all", omitting it.
    console.info node_modules/pino/browser.js:188
      `operatingSystem` filter is set to "all", omitting it.

  ● /home/ajin/Desktop/outreachy/addons-frontend/tests/unit/amo/pages/TestSearchPage.js › dispatches a server redirect when `atype` parameter is "4"

    AssertError: expected dispatch to be called with arguments

      135 |     });
      136 | 
    > 137 |     sinon.assert.calledWith(
          |                  ^
      138 |       fakeDispatch,
      139 |       sendServerRedirect({
      140 |         status: 301,

      at Object.fail (node_modules/sinon/lib/sinon/assert.js:107:21)
      at failAssertion (node_modules/sinon/lib/sinon/assert.js:66:16)
      at Object.calledWith (node_modules/sinon/lib/sinon/assert.js:92:13)
      at Object.<anonymous> (tests/unit/amo/pages/TestSearchPage.js:137:18)
```

Reply:
yes, I will

Reply:
Please add this test case back so that I can investigate in your branch directly.

Reply:
Still breaks, but added it back so you can have a look at it :) @willdurand